src/style.css

```diff
@@ -1 +1,51 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  margin: 0;
+  padding: 0;
```

Comment:
you don't need to repeat styles so for better readability delete all unnecessary styles